clients/admin-ui/src/features/system/system.slice.ts

```diff
@@ -108,8 +108,9 @@ export const {
 export interface State {
   activeSystem?: System;
   activeClassifySystemFidesKey?: string;
+  systemsToClassify: System[];
 }
-const initialState: State = {};
+const initialState: State = { systemsToClassify: [] };
```

Comment:
Not sure if it applies here, but `systemsForReview` initialize as undefined to differentiate between "nothing selected _yet_" and "zero things were selected". The ScanResults component might, for example, distinguish between "no results came back yet" and "empty results" (but maybe the preceeding steps mean it doesn't matter).

Also, does this state ever get cleared? I don't see a `setSystemsToClassify([])` anywhere.

Reply:
hmm that's a good point about it not getting cleared. this page kind of lives in a weird place in the onboarding flow in that it is part of it, but is also its own page so that people can feasibly go back to it (although we don't include it in the nav bar yet). so I'm not sure when the use case for clearing it would be, but I'll keep noodling on it as we add to the feature

and yeah that makes sense about empty vs undefined—I'll update!